api/src/app/modules/submission/submission.service.ts

```diff
@@ -67,6 +67,12 @@ export class SubmissionService extends DataService<Submission, Repository<Submis
 
     const spatialObjects: SpatialObject[] = await this.prepareFomSpatialObjects(submission.id, dto.spatialObjectCode, dto.jsonSpatialSubmission, user);
 
+    // apply simplification algorithm for spatial geometry
+    for (const spatialObject of spatialObjects) {
+      const simplifiedGeometryJson = await this.simplifyGeometry(JSON.stringify(spatialObject.geometry), SpatialCoordSystemEnum.WGS84);
```

Comment:
I wonder which coordinate system code I should use here, in our spike ticket, we use "4326" which WGS84, but in the `convertGeometry`, we convert to BC_ALBERS "3005". However, if I use "3005" here, I'll get an error complaining about the coordinate. 

![image](https://github.com/bcgov/nr-fom/assets/23131735/de86ff91-ca42-405b-abed-bd0cb5dd3df7)

Reply:
You have to use BC Albers - I'd do the coordinate system change (if necessary) first to BC_Albers, then do the simplify. I don't see the context of the error, but for the code fragment we don't want to be looping through the spatial objects (too slow) so perhaps its best just to revert back to doing the update statements.

Reply:
I did encounter the same error.
However, this is outdated. After discussion, Basil would favor for using db update statement for simplification.
PR has been updated.